utils/src/main/kotlin/Utils.kt

```diff
@@ -140,10 +146,35 @@ fun String.fileSystemDecode(): String =
         java.net.URLDecoder.decode(this, "UTF-8")
 
 /**
- * Delete files recursively without failing if e.g. individual files in symlinked directories could not be deleted due
- * to permission issues if in the end the named directory does not exist anymore.
+ * Delete files recursively without following symbolic links (Unix) or junctions (Windows).
```

Comment:
I don't understand how the code below skips symbolic links, could you explain please?

Reply:
It's implicitly documented in the Java SDK. I'll add a code comment.

Reply:
Thanks.